lib/recaptcha/configuration.rb

```diff
@@ -37,6 +37,8 @@ def initialize #:nodoc:
 
       @secret_key           = ENV['RECAPTCHA_SECRET_KEY']
       @site_key             = ENV['RECAPTCHA_SITE_KEY']
+      # Calling @disable will not hit the setter below
+      self.disable          = ['1', 'true', 'yes'].include? ENV['RECAPTCHA_DISABLE']
```

Comment:
maybe simpler to just set `@disabled` instead of extra logic to set and remove an env ... which is also kinda unexpected

my previous comment was to suggest not doing this PR and conditionally adding skip_verify_env from the host app, not doing it here ... sorry for the confusion